src/api/app/assets/stylesheets/webui/application/dialog.scss

```diff
@@ -6,6 +6,10 @@
   height: 10em;
   width: 26em;
   margin-left: -13em;
+  /* TODO: Agree on a common way to use dialogs. Without this adjustment the width of 95% gets applied to the branching dialog and this looks weird */
+  input[name="target_package"] {
```

Comment:
More details can be found here: https://github.com/openSUSE/open-build-service/pull/2720/commits/72423bd777a6445a2dcf77ab28a38dd0b4fdb127